docs/fhir-guide/FHIR 101 - Practical Guide.ipynb

```diff
@@ -935,16 +1061,346 @@
     "        \"profile\": [\"http://fhir.kids-first.io/StructureDefinition/Patient\"]\n",
     "    }\n",
     "}\n",
-    "success, result = client.send_request('POST', endpoint, json=patient_rs_invalid)\n",
-    "assert not success"
+    "validate_rs('Patient', patient_rs_invalid, expected_success=False)"
```

Comment:
As commented above, `validate_rs('Patient', patient_rs_invalid, print_result=True)` so that we can discuss the different messages of success and failure?

Reply:
Second thought: the notebook may get too lengthy. We'd better turn `print_result` on for the actual running demo.